linera-core/src/unit_tests/worker_tests.rs

```diff
@@ -1282,10 +1306,10 @@ where
         None,
     )
     .await;
-    assert!(worker
+    assert!(matches!(worker
         .fully_handle_certificate(certificate, vec![])
-        .await
-        .is_err());
+        .await,
+        Err(WorkerError::ChainError(error)) if matches!(*error, ChainError::InactiveChain{..})));
```

Comment:
@ma2bd: Is this intended?
It looks like it should use `ChainDescription::Root(2)` above, and fail due to the wrong signature instead?

Reply:
Oops. Happy to fix the test after this PR lands.